vagrant/provision-master.sh

```diff
@@ -107,3 +107,23 @@ fi
 # Set up the OPENSHIFTCONFIG environment variable for use by osc
 echo 'export OPENSHIFTCONFIG=/vagrant/openshift.local.config/master/admin.kubeconfig' >> /root/.bash_profile
 echo 'export OPENSHIFTCONFIG=/vagrant/openshift.local.config/master/admin.kubeconfig' >> /home/vagrant/.bash_profile
+
+# wait for openshift-master to come alive
+export OPENSHIFTCONFIG=/vagrant/openshift.local.config/master/admin.kubeconfig
+while true; do
+  if ! /usr/bin/openshift cli get nodes; then
+    sleep 1
```

Comment:
@rajatchopra How about adding an echo -n '.' before the sleep so people will know their waiting and not locked up?